doc/code.html

```diff
@@ -239,28 +250,29 @@ <h3 id="Command">Your first program</h3>
 </pre>
 
 <p>
-Now you can build and install that program with the <code>go</code> tool:
+Now you can build that program with the <code>go</code> tool:
 </p>
 
 <pre>
-$ <b>go install github.com/user/hello</b>
+$ <b>go build ./...</b>
 </pre>
 
-<p>
-Note that you can run this command from anywhere on your system. The
-<code>go</code> tool finds the source code by looking for the
-<code>github.com/user/hello</code> package inside the workspace specified by
-<code>GOPATH</code>.
-</p>
+<p>This will compile your Go code into a <code>./hello</code> binary that can be executed.</p>
+
+<pre>
+$ <b>./hello</b>
+<b>Hello, world.</b>
+</pre>
 
 <p>
-You can also omit the package path if you run <code>go install</code> from the
-package directory:
+Note that you can also use the <code>install</code> command to add your compiled binary
+executable to <code>$HOME/go/bin/hello</code>. 
 </p>
 
 <pre>
-$ <b>cd $GOPATH/src/github.com/user/hello</b>
-$ <b>go install</b>
+$ <b>go install github.com/user/hello</b>
+$ <b>$HOME/go/bin/hello</b>
+hello, world.
```

Comment:
Looks like we run the code further down in the tutorial, don't need it twice

Reply:
Done, you may have to comment "I signed it!" though, looks like the original attempt may have failed.